Languages/french.lang

```diff
@@ -550,22 +550,22 @@ msgid "Current neek files are not neek2o. Game autoboot disabled."
 msgstr "Le fichier neek actuel n'est pas neek2o. Démarrage automatique désactivé."
 
 msgid "Custom Address"
-msgstr ""
+msgstr "Adresse perso"
```

Comment:
```suggestion
msgstr "Adresse personnalisé"
```

I propose this to render _custom_ better: it’s customized meaning non-default, not personal (perso = personnel to me)

Reply:
personnalisée is better :-)